library/L1_Peripheral/lpc40xx/eeprom.hpp

```diff
@@ -94,18 +99,19 @@ class Eeprom final : public sjsu::Eeprom
   void Write(const uint8_t * wdata, uint32_t full_address,
              size_t count) const override
   {
-    constexpr uint8_t kMax6Bits = 0b11'1111;
+    bit::Mask k_lower6_bits = bit::CreateMaskFromRange(0, 5);
+    bit::Mask k_upper6_bits = bit::CreateMaskFromRange(6, 11);

```

Comment:
Do no proceed a variable name with a k if it is not a constant. This should be a constant and should be Cap Camel case with a lowercase k in front.